config/config.go

```diff
@@ -76,7 +76,8 @@ type (
 		MaxPeers          int                      `yaml:"MaxPeers"`
 		AttemptConnPeers  int                      `yaml:"AttemptConnPeers"`
 		MinPeers          int                      `yaml:"MinPeers"`
-		Monitoring        metrics.PrometheusConfig `yaml:"Monitoring"`
+		Monitoring        metrics.Config           `yaml:"Monitoring"`
```

Comment:
It just occured to me that now this name is probably too ambiguos, even in README it's not obvious what kind of monitoring we're configuring here. Maybe rename it to `Prometheus` now?